noggin/defaults.cfg

```diff
@@ -5,6 +5,9 @@ SESSION_COOKIE_HTTPONLY = True
 SESSION_COOKIE_SECURE = True
 USER_DEFAULTS = {"user_locale": "en-US", "user_timezone": "UTC"}
 THEME = "default"
+PASSWORD_POLICY = {
+    "min": 8, "max": -1
+}
```

Comment:
If the password policy is changed on the freeIPA side does it also have to be manually changed here on the Noggin side?

If so, i think we either need to get the password policy from IPA (if we can), or document this really well for system admins so we dont run into issues where the password policies dont match.

Reply:
Yeah, I agree. There is a way to get the generic password policy from IPA or group-specific password policies, but I don't think Noggin has a way to determine which policy is going to be used for the users that it creates. Or maybe it does but I'm betting that would be pretty complex.
At first I wanted to let FreeIPA errors propagate back to the user but unfortunately FreeIPA only tells you that your password does not match the policy, it does not tell you what the policy is, so the user is left wondering what they have done wrong. I think it's less bad to ask ops to adjust the password policy in noggin when they change it in FreeIPA. Maybe there's a way to do it better but I think it'll do for now (I'll open a ticket for that when this is merged so we remember it).